pretix_eth/payment.py

```diff
@@ -50,15 +50,17 @@ def settings_form_fields(self):
         d.move_to_end('DAI', last=True)
         return d
 
-    def is_allowed(self, request, total):
-        return self.settings.DAI or self.settings.ETH and super().is_allowed(request, total)
+    def is_allowed(self, request):
+        return bool(
+            (self.settings.DAI or self.settings.ETH) and super().is_allowed(request)
+        )
 
     @property
     def payment_form_fields(self):
         e = ('ETH', _('Ethereum'))
         d = ('DAI', _('DAI'))
         if self.settings.ETH and self.settings.DAI:
-            tup = (e, d)
+            tup = (d, e)
```

Comment:
Why does order matter here? Maybe deserves a comment.

_Edit: Ah, maybe it affects the order it shows up in the payment form?_

Reply:
Yeap, that's what order represents there.  And maybe just to keep things alphabetical in the form?